apps/vr-tests-web-components/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "ESLINT_USE_FLAT_CONFIG=false eslint src --ext .ts,.tsx",
     "start": "storybook dev",
     "type-check": "tsc -p . --baseUrl . --noEmit",
-    "test-vr": "storywright  --browsers chromium --url dist/storybook --destpath dist/screenshots --waitTimeScreenshot 500 --concurrency 4 --headless true"
+    "test-vr": "storywright --browsers chromium --url dist/storybook --destpath dist/screenshots --waitTimeScreenshot 500 --concurrency 4 --headless true --stepsApi parameters"
   },
   "dependencies": {
     "react": "18.3.1",
```

Comment:
with new platform agnostic api from storywright WC will no longer need react for authoring stories @chrisdholt @radium-v ;)